depends/packages/native_capnp.mk

```diff
@@ -0,0 +1,17 @@
+package=native_capnp
+$(package)_version=0.7.0
+$(package)_download_path=https://capnproto.org/
+$(package)_file_name=capnproto-c++-$($(package)_version).tar.gz
```

Comment:
IIRC the "+" in the filename gives some tools/filesystems grief, which is why we hack it out in native_cctools.
You can use (untested):
```
$(package)_download_file=capnproto-c++-$($(package)_version).tar.gz
$(package)_file_name=capnproto-c++-$($(package)_version).tar.gz
```
I'll try to track down a good explanation for why we do that first, though. Maybe it's no longer necessary.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16367#discussion_r388486877

> IIRC the "+" in the filename gives some tools/filesystems grief, which is why we hack it out in native_cctools.

Thanks. It'd be nice to not have to do this if you find out it's unnecessary, but replaced + with x for now